jabgui/src/main/java/org/jabref/gui/ai/components/aichat/AiChatComponent.java

```diff
@@ -410,4 +424,63 @@ private Optional<UserMessage> getLastUserMessage() {
         }
         return Optional.empty();
     }
+
+    @FXML
+    private void exportMarkdown() {
+        if (aiChatLogic.getChatHistory().isEmpty()) {
+            dialogService.notify(Localization.lang("No chat history to export"));
+            return;
+        }
+
+        FileDialogConfiguration fileDialogConfiguration = new FileDialogConfiguration.Builder()
+                .addExtensionFilter(StandardFileType.MARKDOWN)
+                .withDefaultExtension(StandardFileType.MARKDOWN)
+                .withInitialDirectory(Path.of(System.getProperty("user.home")))
+                .build();
+
+        dialogService.showFileSaveDialog(fileDialogConfiguration)
+                     .ifPresent(path -> {
+                         try {
+                             AiExporter exporter = new AiExporter(entries.getFirst(), entryTypesManager, fieldPreferences);
+                             String content = exporter.buildMarkdownForChat(aiChatLogic.getChatHistory());
+                             Files.writeString(path, content, StandardOpenOption.CREATE, StandardOpenOption.TRUNCATE_EXISTING);
+                             dialogService.notify(Localization.lang("Export successful"));
+                         } catch (IOException e) {
+                             LOGGER.error(Localization.lang("Problem occurred while writing the export file"), e);
+                             dialogService.showErrorDialogAndWait(Localization.lang("Save failed"), e);
```

Comment:
Why unspecific here? - I think `"Problem occurred while writing the export file"` is the better text. 

However, please check `JabRef_en.properties` for other error messages. Maybe you find some with `%0` included, where `ex.getLocaliizedMessage()` is passed to.